### PR TITLE
Issue 23058 listing favorite pages not refreshing after saved

### PR DESCRIPTION
**Pull request by alfredo-dotcms**
### Proposed Changes
* fix listing favorite pages not refreshing after saved
* fix actions in context menu not matching same order from Content Search

### Checklist
- [ ] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
https://github.com/dotCMS/core/issues/23058#issuecomment-1444537929

### CODE REVIEW

File: core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-store/dot-pages.store.ts [hunk 1]
```diff
@@ -469,6 +531,14 @@ export class DotPageStore extends ComponentStore<DotPagesState> {
             });
         }
```
Comment (alfredo-dotcms) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-store/dot-pages.store.ts:533]: fix actions in context menu not matching same order from Content Search

File: core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-store/dot-pages.store.ts [hunk 2]
```diff
@@ -386,6 +431,34 @@ export class DotPageStore extends ComponentStore<DotPagesState> {
...
+            query: this.getESQuery(),
+            sortField: sortField || 'modDate',
+            sortOrder: sortOrderValue
+        });
+    };
+
+    private formatPagesData = (items: ESContent, offset: number) => {
+        const dateFormattedPages = this.formatRelativeDates(items.jsonObjectView.contentlets);
+
+        let currentPages = this.get().pages.items;
```
Comment (fmontes) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-store/dot-pages.store.ts:451]: Can you sent this from UI?

Comment (fmontes) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-store/dot-pages.store.ts:451]: Can you pass this from UI?

File: core-web/apps/dotcms-ui/src/app/view/components/dot-contentlet-editor/components/dot-contentlet-wrapper/dot-contentlet-wrapper.component.ts [hunk 3]
```diff
@@ -86,8 +85,10 @@ export class DotContentletWrapperComponent {
                         this.dotIframeService.reload();
                     }
 
-                    this.dotEventsService.notify<DotGlobalMessage>('dot-global-message', {
-                        value: this.dotMessageService.get('message.content.saved')
+                    // Message emitted to notify DotPagesComponent
+                    this.dotEventsService.notify('save-page', {
+                        value: this.dotMessageService.get('message.content.saved'),
+                        retryLoading: true
```
Comment (fmontes) [inline core-web/apps/dotcms-ui/src/app/view/components/dot-contentlet-editor/components/dot-contentlet-wrapper/dot-contentlet-wrapper.component.ts:91]: When don't we need this?

**Review by fmontes — CHANGES_REQUESTED** [review]

**Pull request closed without merging**